Task

idwrz opened the pull request.

idwrz committed:
Update حالت استثناء.xml

idwrz committed:
Update حالت استثناء.xml

idwrz committed:
Delete سیستم تبلیغات تسک سیستم.xml

idwrz committed:
Update رزرو تبلیغات 🧸.xml

idwrz committed:
Update تغییر زبان ربات.xml

idwrz committed:
Update Cloudy sky 🌨.xml

idwrz committed:
Update درباره پیام ثبت بازی 🍕.xml

idwrz committed:
Update درباره پیام ثبت بازی 🍕.xml

idwrz committed:
Update پیام اول بعد از ثبت بازی.xml

idwrz committed:
Update پیام دوم بعد از ثبت.xml

idwrz committed:
Update اخرین پیام مربوط به ثبت بازی.xml

idwrz committed:
Update جایزه روزانه 🎁.xml

idwrz committed:
Update جایزه روزانه 🎁.xml

idwrz committed:
Update حالت استثناء.xml

idwrz committed:
Update حالت استثناء.xml

idwrz committed:
Update چجوری کشتنت؟.xml

immmdreza commented:
سلام

idwrz committed:
Update پُر کننده بازی.xml

immmdreza commented on Trees/Farsi/admin_exp/حالت استثناء.xml:
```diff
@@ -1,27 +1,25 @@
-  
\ No newline at end of file
+⚠️ توجه داشته باشید قابلیت مخصوص کریتور گروه بوده و در حال حاضر فقط هفت نفر را میتوان به لیست حالت های استثناء افزود.
+</value>
```
شهلا اخر همه این فایلا باید </item> باشه. الان این نداره

idwrz replied:
عه جا افتاده صاری

immmdreza replied:
اونارو درس کردم

idwrz committed:
Update حذف اکانت امتیازی.xml

idwrz committed:
Delete ◊ چطوری رنک بگیریم ؟.xml

دو تا مقاله مشابهش داریم و کاملتر.

idwrz committed:
Update مدیریت لینک.xml

immmdreza committed:
Fixes some tags missing

immmdreza committed:
Merge branch 'main' of https://github.com/idwrz/InBot-Documentations into pr/3

idwrz committed:
Update کانال ثبت وقایع.xml

idwrz committed:
Update مدیریت رنکها.xml

به اون دو تا قسمت مربوط ب رنک در صورت یکی شدن اضافه شه🌙

idwrz committed:
Update امتیاز شخصی (۱).xml

idwrz committed:
Update دستورات امتیاز شخصی.xml

idwrz committed:
Update رنک ها ۱.xml

idwrz committed:
Delete رنک ها ۲.xml

هر دو برای یه شخصه و چون کوتاه و مربوط به همه، توی یک مقاله‌ در رنک‌ها جا دادیم.

idwrz committed:
Update رنک ها ۴.xml

idwrz committed:
Delete رنک ها ۳.xml

اضافس توضیحاتش رو داریم.

idwrz committed:
Update وزن نقش ها ۱.xml

idwrz committed:
Update وزن نقش ها ۲.xml

idwrz committed:
Update ذخیره نقش ها بخش اول.xml

عمو کلاه آ رو بزار خب😂

idwrz committed:
Update ذخیره نقش ها بخش دوم.xml

idwrz committed:
Update ذخیره نقش ها بخش ۳.xml

idwrz committed:
Update ذخیره نقش ها بخش ۴.xml

idwrz committed:
Update فصل های ربات امتیاز.xml

idwrz committed:
Update ذخیره لینک گروه.xml

دکمه‌اش از توی دکمه‌های اصلی برداشته شه به مدیریت لینک اضافه بشه و با هم یه دکمه‌ی اصلی رو تشکیل بدن اسمش هم میشه باشه ثبت لینک گروه
یعنی دوتا مقاله رو توی دو تا دکمه‌ی شیشه ای با اسم خودشون داشته باشیم اما اون دوتا به جای اینکه هر کدوم جدا جدا در منوی اصلی باشن، با هم دکمه‌ی شیشه ای بسازن

idwrz committed:
Update صدا زدن اعضا.xml

idwrz committed:
Update وب سرویس.xml

idwrz committed:
Update پیام خوش آمد ( نسخه ساده ).xml

idwrz committed:
Update پیوستن به رقابت ۱.xml

idwrz committed:
Update پیوستن به رقابت ۲.xml

idwrz committed:
Update پیوستن به رقابت ۳.xml

idwrz committed:
Update آمار ورولف.xml

immmdreza committed:
Fixes and build jobs.

immmdreza committed:
Added authors's names whom their docs were removed.

immmdreza committed:
Format things

immmdreza committed:
Another build

immmdreza commented:
فکر کنم آمادس

idwrz commented:
هورااا

On Sat, 11 Jun 2022, 16:06 MohammadReza, ***@***.***> wrote:

> فکر کنم آمادس
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/Task-System/InBot-Documentations/pull/3#issuecomment-1152904119>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AZR6MEQAHLCPAMYOFI654JTVOR233ANCNFSM5YMXX7JQ>
> .
> You are receiving this because you were assigned.Message ID:
> ***@***.***>
>
